enclave-tls/README.md

```diff
@@ -0,0 +1,29 @@
+# enclave-tls
+
+## Build
+
+### Build Requirements
+
+- git
+- make
+- autoconf
+- automake
+- libtool
+- go version 1.14 or higher
+
+### Build and Install
+
+```shell
+cd inclavare-containers/enclave-tls
+make
+make install
+```
+
+## Run
+
+```shell
+cd inclavare-containers/enclave-tls/bin
+mkdir -p /run/enclave-tls
+./ra-tls-server run &
+./elv echo
+```
```

Comment:
cd /opt/enclave-tls/bin to run everything instead.

Reply:
Oops. What I talked is supported only in my branch. I will push the codes later.